lib/src/emoji_cell.dart

```diff
@@ -99,16 +101,39 @@ class EmojiCell extends StatelessWidget {
     );
   }
 
-  /// Build and display Emoji centered of its parent
-  Widget _buildEmoji() {
-    final style = TextStyle(
+  static String? _getPlatformDefaultEmojiFont() {
+    if (Platform.isIOS || Platform.isMacOS) {
+      return 'Apple Color Emoji';
+    }
+    return null;
+  }
+
+  TextStyle _getEmojiTextStyle() {
+    final defaultStyle = TextStyle(
       fontSize: emojiSize,
+      // If the user provided a style, don't override the font family.
+      fontFamily: textStyle == null ? _getPlatformDefaultEmojiFont() : null,
+      // Commonly available fallback fonts.
+      fontFamilyFallback: [
+        // iOS and MacOs.
+        'Apple Color Emoji',
+        // Android, ChromeOS, Ubuntu and some other Linux distros.
+        'NotoColorEmoji',
+        // Windows.
+        'Segoe UI Emoji',
+      ],
```

Comment:
That works nicely for the emoji picker. But we need to do similar thing for the textInput. Maybe we can expose this as a textStyle that can be reused for TextInput. I checked it quickly and some emoji look fine, like numbers, but other like M and O they appear differently, not sure what causing it. 
![Simulator Screen Shot - iPhone 8 - 2023-11-30 at 22 53 09](https://github.com/Fintasys/emoji_picker_flutter/assets/11667234/c010d543-e797-49b2-aba2-80c8d41bf4f7)

Reply:
Yeah that's a good point. Maybe we can expose it through config.dart and use it as the default text style when the user hasn't specified one. Let's resolve the other questions first and then move this and apply it to the text input.